RTCPeerConnection.js

```diff
@@ -230,6 +259,10 @@ export default class RTCPeerConnection extends EventTarget(PEER_CONNECTION_EVENT
     return this._remoteStreams.slice();
   }
 
+  getTranscievers() {
```

Comment:
Is it what you want to call Transcievers instead Transceivers or a mistake? I catch some other places where name looks like wrong.

Reply:
Thanks! I totally miss this. For some reason this word give me a headache and i have made dozen of typos while doing this PR :)

Should be fixed now.